docker/pipelines_runner.sh

```diff
@@ -88,7 +110,7 @@ function main {
     --machine-type "g1-small" \
     --pvm-attempts 0 \
     --attempts 1 \
-    --disk-size 10
+    --disk-size 10 ${extra_args}
```

Comment:
I did a quick check and it seems Dataflow has `--use_public_ips` (or ` --no_use_public_ips`). 
So if we want to do what I suggested we need to construct two `extra_args`: one for Dataflow one for pipelines-tool.
If you think this makes things more complex and does not help with users please feel free to ignore.

Reply:
Yes, unfortunately it is different and per my other comment is it even different for Java vs. Python.

I didn't add them or do the extra parsing since users are putting much of the Dataflow command together already.  That said, I would be happy to add a second optional set of arguments to the script for Dataflow if we can resolve the Java vs. Python issue.

Reply:
In the example usage I have seen the user is needing to build up a fair bit of the Dataflow command as it is, so it is easy for users to assume they can extra arguments to the Dataflow command as they need.  I think it would be nice to pass `temp_location`, `subnetwork` and `use_public_ip`, however, it can be confusing.

In fact at first not understanding the flow of Container -> VM -> Dataflow and the arguments made it tricky to understand what the problem was in the case of having a custom network because we thought having the subnet flag in the Dataflow command was enough.

Reply:
Thanks for the feedback. So based on your suggestion #622 is a good idea.

After your PR merged we can add a small section to our docs and explain these two new flags. In that example we can show these flags need to be added to both Dataflow args and launcher VM args. 
If you will, I'd appreciate if you help us with that change too. Specially the wording related to these requirements to be HIPAA compliant.

Reply:
Yes, I think it makes things easier, so I'll add some logic to this PR for setting the corresponding Dataflow flags.

Also happy to help with other PR.